src/Agent.Sdk/DockerVersion.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Agent.Sdk
+{
+    public class DockerVersion
+    {
+        public DockerVersion()
```

Comment:
Do we need the empty constructor?

Reply:
not 100% sure on this one, this was just a code relocation, but in general, we need empty constructors for objects that get serialized to JSON if there is a non-empty constructor.